common/static/js/capa/src/formula_equation_preview.js

```diff
@@ -139,8 +139,7 @@ formulaEquationPreview.enable = function () {
                         if (inputData.jax) {
                             // Set the text as the latex code, and then update the MathJax.
                             MathJax.Hub.Queue(
-                                ['Text', inputData.jax, latex],
-                                ['Reprocess', inputData.jax]
+                                ['Text', inputData.jax, latex]
```

Comment:
Setting the Text will also cause it to reprocess. Maybe it didn't in any earlier version?